packages/runtime-core/src/apiAsyncComponent.ts

```diff
@@ -113,6 +113,8 @@ export function defineAsyncComponent<
     name: 'AsyncComponentWrapper',
     setup() {
       const instance = currentInstance!
+      let delayTimer: number | null = null
```

Comment:
you can remove the initialization in both
```suggestion
      let delayTimer: number | null | undefined
```

Reply:
done